src/command/helpModule.ts

```diff
@@ -39,7 +40,25 @@ export default class HelpModule extends Module {
                 true
             );
         }, initialEmbed);
-
-        await msg.channel.send(embed);
+        
+        const nonEmbed = `${CODEBLOCK}
```

Comment:
Rename to `fallbackContent` and maybe define this inside the scope of the `if` block instead.